mon-pix/app/routes/user-tutorials-v2/saved.js

```diff
@@ -4,11 +4,32 @@ export default class UserTutorialsSavedRoute extends Route {
   @service featureToggles;
   @service router;
 
+  queryParams = {
+    pageNumber: { refreshModel: true },
+    pageSize: { refreshModel: true },
+  };
+
   redirect() {
     if (!this.featureToggles.featureToggles.isNewTutorialsPageEnabled) this.router.replaceWith('user-tutorials');
   }
 
-  model() {
-    return this.store.query('tutorial', { type: 'saved' }, { reload: true });
+  model(params) {
+    return this.store.query(
+      'tutorial',
+      {
+        type: 'saved',
+        page: {
+          number: params.pageNumber,
+          size: params.pageSize,
+        },
+      },
+      { reload: true }
+    );
+  }
+
+  resetController(controller, isExiting) {
+    if (isExiting) {
+      controller.set('pageNumber', 1);
```

Comment:
```suggestion
      controller.set('pageNumber', null);
```
Suggestion: Je pense que cela peut être fait dans cette PR dans un nouveau commit. Le passage de la valeur `1` à `null` dans ce fichier ainsi que dans `recommended.js` corrige le bug remonté par @yannbertrand [ici](https://github.com/1024pix/pix/pull/4196#issuecomment-1067051160)
(Ajouter un test pour confirmer cela dit)